tests/Model/IndexInfoFunctionalTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace MongoDB\Tests\Model;
+
+use MongoDB\Collection;
+use MongoDB\Operation\CreateIndexes;
+use MongoDB\Operation\ListIndexes;
+use MongoDB\Tests\Operation\FunctionalTestCase;
+
+class IndexInfoFunctionalTest extends FunctionalTestCase
+{
+    private $collection;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->collection = new Collection($this->manager, $this->getDatabaseName(), $this->getCollectionName());
```

Comment:
Since we're creating a collection in each test, let's drop it upon `setup()` and tearDown()`. Consider `MongoDB\Tests\Collection\FunctionalTestCase` for prior art. In particular, we can leave the collection as-is in the event of a failure.

I see no reason we can't use the Collection methods, so `$this->collection->drop();` is fine and we can skip the DropCollection operation class.